core/src/main/java/brooklyn/location/access/PortForwardManagerImpl.java

```diff
@@ -38,12 +25,25 @@
 import brooklyn.location.basic.AbstractLocation;
 import brooklyn.mementos.LocationMemento;
 import brooklyn.util.collections.MutableMap;
-
 import com.google.common.base.Objects.ToStringHelper;
+import com.google.common.base.Predicate;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Lists;
 import com.google.common.net.HostAndPort;
+import org.slf4j.Logger;
```

Comment:
I'm finding inconsistencies in how Eclipse organises the imports. Configuring IntelliJ to have the same imports list as Eclipse unfortunately doesn't work. As @neykov put it: "One of the IDEs (don't remember which) was interleaving packages not specified in the list, while the other was putting them at the start and then listing the remaining packages."